src/components/scene/canvas.js

```diff
@@ -14,42 +20,40 @@ module.exports.Component = register('canvas', {
     document.addEventListener('mozfullscreenchange', onFullScreenChange);
     document.addEventListener('webkitfullscreenchange', onFullScreenChange);
 
-    // Prevent overscroll on mobile.
-    canvasEl.addEventListener('touchmove', function (event) {
-      event.preventDefault();
+    // Listen to mousedown to enable grab/grabbing cursors.
+    canvasEl.addEventListener('mousedown', function addGrabbingClass (event) {
+      document.body.classList.add(GRABBING_CLASS);
+    });
+    document.body.addEventListener('mouseup', function removeGrabbingClass (event) {
+      document.body.classList.remove(GRABBING_CLASS);
     });
 
-    // Handle fullscreeen styling
-    sceneEl.addEventListener('enter-vr', addFullscreenClass);
-    sceneEl.addEventListener('exit-vr', removeFullscreenClass);
+    // Handle fullscreeen styling.
+    sceneEl.addEventListener('enter-vr', function addFullscreenClass () {
```

Comment:
I think moving it down connotes that it's reused multiple times. And when they're separated from where they're used, it takes effort to see where it's being used.